src/styles/components/pages/Offerer/BankInformation/_BankInformation.scss

```
@@ -0,0 +1,43 @@
+.bank-information {
+  position: relative;
+
+  .bi-subtitle {
+    color: $grey;
+    margin: 16px 0;
```

```suggestion
    margin: rem(16px) 0;
```
Pour pouvoir zoomer textuellement. Je vous laisse corriger les autres.